src/index.js

```diff
@@ -4,7 +4,6 @@ import './style.css';
 
 //Reference HTML
 const myInput = document.querySelector('input');
-// eslint-disable-next-line no-unused-vars
 const clearContent = document.querySelector('#clear-content');
 const todoList = document.querySelector('#display-list');
 
```

Comment:
- [ ]  You are almost there! There's an bug with the remove/clear functionality, after removing some tasks if you add anew one, the removed ones reappear:

| 1. Add tasks|2. Remove some tasks| 3. Add a new one| 4. Removed tasks appear again|
|---|---|---|---|
| ![Captura de pantalla 2022-06-03 a la(s) 10 35 29 p  m](https://user-images.githubusercontent.com/83717285/171981331-0492f39e-32b1-4889-bdd9-a73ad79dcca2.png)|![Captura de pantalla 2022-06-03 a la(s) 10 35 52 p  m](https://user-images.githubusercontent.com/83717285/171981383-0efb049d-0044-469d-8f39-ec04fcef3be3.png)|![Captura de pantalla 2022-06-03 a la(s) 10 36 06 p  m](https://user-images.githubusercontent.com/83717285/171981404-6e790c3d-e9a2-4980-b073-c4f5d8f870bb.png)|![Captura de pantalla 2022-06-03 a la(s) 10 54 59 p  m](https://user-images.githubusercontent.com/83717285/171981408-bd864c6e-fbc6-4982-a10c-aed7907ada45.png)|

- [ ] It would be great to fix that bug, I would suggest checking if the localStorage and the tasks array are being updated after one task is removed.

Reply:
@smunozmo ,This is now fixed. Thank you for the correction